confluent_platform/assets/service_checks.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "agent_version": "6.19.0",
+        "integration": "Confluent Platform",
+        "groups": [
+            "host",
+            "instance"
+        ],
+        "check": "confluentplatform.can_connect",
```

Comment:
Typo?

```suggestion
        "check": "confluent_platform.can_connect",
```

Reply:
Not a typo, currently, jmxfetch is stripping out `_`: https://github.com/DataDog/jmxfetch/blob/10c216bf3bc9ebfba368780fbb16ba05d05fe6e4/src/main/java/org/datadog/jmxfetch/reporter/Reporter.java#L185-L189